### PR TITLE
Topology support for VO scitokens credentials

### PR DESCRIPTION
**Pull request by matyasselmeci**
https://opensciencegrid.atlassian.net/browse/SOFTWARE-4453

### CODE REVIEW

File: virtual-organizations/GLOW.yaml [hunk 1]
```diff
@@ -28,6 +28,10 @@ Contacts:
   VO Manager:
   - ID: 48e1c2f26dc3479a6cf9b2de7c79d654ac27b1d1
     Name: Miron Livny
```
Comment (brianhlin) [inline virtual-organizations/GLOW.yaml:30]: Separately, should this be @lmichael107 ?

File: virtual-organizations/GLOW.yaml [hunk 2]
```diff
@@ -28,6 +28,10 @@ Contacts:
...
+Credentials:
+  TokenIssuers:
+    - URL: https://scitokens.org/chtc
```
Comment (brianhlin) [inline virtual-organizations/GLOW.yaml:33]: Is this actually going to be the GLOW issuer?

**Review by brianhlin — COMMENTED** [review]
Maybe we should go with `DefaultUnixUser` or `DefaultUser`? Some other minor questions

File: virtual-organizations/GLOW.yaml (hunk 2 above)
Comment (matyasselmeci) [reply virtual-organizations/GLOW.yaml:33]: 🤷🏻 It's what we used for StashCache below.

File: virtual-organizations/GLOW.yaml (hunk 1 above)
Comment (matyasselmeci) [reply virtual-organizations/GLOW.yaml:30]: I have no idea what these fields even mean...

Comment (brianhlin) [reply virtual-organizations/GLOW.yaml:30]: We mainly use contact lists across VOs/sites to determine who to contact for various emails. Site contacts are increasingly being used to authorize token requests (i.e., CE and OS registries)

**Comment by brianhlin** [comment]
> Maybe we should go with DefaultUnixUser or DefaultUser?

@matyasselmeci bikeshedding question from above

**Comment by matyasselmeci** [comment]
> > Maybe we should go with DefaultUnixUser or DefaultUser?
> 
> @matyasselmeci bikeshedding question from above

I definitely want to keep Unix in there -- "User" can mean so many things. 50-50 on Default; do we have to remind people here that they can change it?

**Comment by brianhlin** [comment]
> do we have to remind people here that they can change it?

I certainly don't think it hurts. It's similar to how we call the VO mapping file `voms-mapfile-default`

**Comment by matyasselmeci** [comment]
> > do we have to remind people here that they can change it?
> 
> I certainly don't think it hurts. It's similar to how we call the VO mapping file `voms-mapfile-default`

OK, made the change. Anything else?

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**